soh/src/code/z_player_lib.c

```diff
@@ -6,6 +6,8 @@
 #include "objects/object_triforce_spot/object_triforce_spot.h"
 #include "overlays/actors/ovl_Demo_Effect/z_demo_effect.h"
 
+#include "z64.h"
+
 #include "soh/Enhancements/game-interactor/GameInteractor.h"
```

Comment:
Comes with `global.h`

```suggestion
#include "soh/Enhancements/game-interactor/GameInteractor.h"
```